### PR TITLE
Rm yoda news/v3.0.x

### PR DESCRIPTION
**Pull request by artpol84**

### CODE REVIEW

**Comment by bwbarrett** [comment]
Thanks for the PR; can you also file a PR against master for the same updates?

**Comment by artpol84** [comment]
I can, but master's NEWs seems outdated

**Comment by artpol84** [comment]
https://github.com/open-mpi/ompi/pull/3923 is merged, so this changes are in the master already.

File: NEWS [hunk 1]
```diff
@@ -68,6 +68,12 @@ Major new features:
 Changes in behavior compared to prior versions:
 
 Removed legacy support:
+- "yoda" SPML component is removed from the Open SHMEM codebase. It was utilizing
+  Open MPI BTL components for supported networks.
+  Currently avalable SPML components are:
```
Comment (hppritcha) [inline NEWS]: I'd rather this just include the comment about yoda removal, not the other items.  It makes the removed legacy support section harder to read.

Comment (artpol84) [inline NEWS]: I can do that, My intention was to provide alternatives for those who was relying on yoda.

**Comment by ibm-ompi** [comment]
The IBM CI (XL Compiler) build failed! Please review the log, linked below.

 Gist: https://gist.github.com/ibm-ompi/88b480fb98dfd2bdac939f023e1f8fe6

**Comment by jjhursey** [comment]
bot:ibm:xl:retest

**Comment by hppritcha** [comment]
:+1:

**Pull request merged**